### PR TITLE
docs: fix typos in README files

### PR DESCRIPTION
**Pull request by ShatilKhan**
All typos addressed in this issue #311 have been fixed.
Had come across some problems while making the PR , it was taking in previous commits as well. But I managed to get it fixed.
The Issue was very detailed, so it helped a lot in making the changes.

### CODE REVIEW

File: packages/api/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ npm start
 
 To test Stripe integration, you need to create a Stripe account and get the API keys.
 
-Then, you need to add in your `packages/api/.env` file in the following content:
+Then, you need to add your `packages/api/.env` file in the following content:
```
Comment (anfibiacreativa) [inline packages/api/README.md:36]: Hi, thank you for you contribution. 

I think this should read:

`Then, you need to add in the following variables in your `packages/api/.env` file :`

File: packages/docs/README.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ This is the "developer guide" documentation package for the Contoso Real Estate
 
  - **`website/`** | source for the _static website_ hosting a developer guide for self-guided exploration of documentation from _design_ to _deployment_ steps. See [website/README](website/README.md) for details.
  
- - **`training/`** | content for _interactive workshops_ to explore sample in a hands-on, step-by-step manner ex: _Learn Live_ events. See [training/README](website/README.md).
+ - **`training/`** | content for _interactive workshops_ to explore the sample in a hands-on, step-by-step manner ex: _Learn Live_ events. See [training/README](website/README.md).
```
Comment (anfibiacreativa) [inline packages/docs/README.md:12]: I think `the samples` in plural is even more accurate. 
Thank you!

File: packages/api/README.md (hunk 1 above)
Comment (ShatilKhan) [reply packages/api/README.md:36]: Yeah I'll make sure to change it.
Thanks @anfibiacreativa ✨

**Comment by ShatilKhan** [comment]
Please check my updated commit @anfibiacreativa 
Let me know if there are any changes required.

**Review by anfibiacreativa — DISMISSED** [review]
Thanks for your contribution!

**Comment by anfibiacreativa** [comment]
@ShatilKhan  I see you reverted the changes in commit https://github.com/Azure-Samples/contoso-real-estate/pull/312/commits/68ea122e509f78a729eea2b2d5b29a29585d8365

**Comment by ShatilKhan** [comment]
Sorry for the confusion guys, check this PR #310
This PR only fixes some Typos

**Review by anfibiacreativa — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**